### ISSUE TITLE
Options page: not working with screen readers (and odd behavior of headers)

### ISSUE THREAD

**Issue by jeffbl** [Shared-Reality-Lab/IMAGE-browser#416]
While doing training for NFB yesterday, we ran into trouble with the options panel. @VenissaCarolQuadros and I were watching someone's shared screen while they were trying to enable the Monarch option in Options. Observations:

1. The headers trigger and expand/contract all the time, and don't work correctly even with a mouse (e.g., sometimes I click while they are in one state, and they stay in that state until I click them again).
SUGGEST: Remove expansion/contraction of the headers, and just have them all expanded all the time, so those headers just become non-interactive elements.

2. When the Haptic Devices section was expanded, the Monarch label was visible, but not the toggle switch. The user could not seem to tab to the control and hear it in the screen reader. We're not sure why he could not "find" it. He mentioned that he was hearing the developer mode twice, which could indicate the toggle is mis-labeled for screen readers? In any event, it was not accessible. With multiple painful minutes of fiddling, somehow it turned on, but we have no idea how it finally triggered. Once it was expanded, the fields for the codes and everything worked fine.

Technical setup for user:
Windows
JAWS screen reader
Some sort of high contrast dark mode was enabled (e.g., background black, all controls white)

In general, we should make sure we're using very simple, straightforward layouts, and standard controls, and double-check all labels, since screen readers can do funky things, and users will have a variety of accessibility options set.

**Comment by jeffbl** [comment]
If fixes seem safe enough, I think we should try and push an update asap, so that if Greg or anyone needs to re-enable this while using a screen reader, it consistently works. Code should be reviewed by @JRegimbal before merge, as an additional check since we're so close to NFB.
